code/Jen/javascript/Lab_06/quoteAPI.js

```diff
@@ -0,0 +1,68 @@
+
+
+const App = {
+
+    el: '#app',
+    data() {
+        return {
+            output: {},
+            search: {},
+
+        }
+
+    },
+
+    methods: {
+
+        getQotd() {
+
+
+
+            console.log('GET Request');
+            axios.get('https://favqs.com/api/qotd')
+                .then(res => {
+                    console.log(res)
+                    this.output = res.data.quote.body
+
+                })
+                .catch(err => console.error(err))
+
+        },
+
+        getQuote(search) {
+            console.log(search)
+            axios.get(`https://favqs.com/api/quotes/`,
+                { headers: { "Authorization": "Token token = '855df50978dc9afd6bf86579913c9f8b'" }, params: { "filter": search } })
```

Comment:
the spaces around the `=` that's inside the authorization key are the problem. The call works with `'Authorization': 'Token token="855df50978dc9afd6bf86579913c9f8b"'`

It's because this weird string value is being parsed on the backend so it has to be character-for-character perfect.